src/Lib/Managers/SelectionManager.vala

```diff
@@ -51,14 +51,21 @@ public class Akira.Lib.Managers.SelectionManager : Object {
     }
 
     public Lib.Items.NodeSelection selection;
+    public int? alignment_anchor_id;
```

Comment:
Maybe we should give this a more generic name and keep it open for potentially other features? Perhaps calling it subselection?

Reply:
To me it's ok, this was only intended for the alignment anchor thing, but I can change it to `sub_selection_id`.